setup/rhel72-linuxonecc/resources/jenkins.service.j2

```diff
@@ -12,7 +12,7 @@ User={{ server_user }}
 Environment="USER={{ server_user }}" \
             "SHELL=/bin/bash" \
             "HOME=/data/{{ server_user }}" \
-            "PATH=/usr/lib64/ccache:/usr/bin" \
+            "PATH=/usr/local/bin:/usr/bin" \
```

Comment:
Because the ansible task we have for compiling/installing ccache manually installs to /usr/local/bin.